corehq/apps/translations/integrations/transifex/parser.py

```diff
@@ -12,7 +12,7 @@
     # Module or Form: sheet name for module/form: unique id
     'module_and_forms_sheet': r'^(Module|Form):(\w+):(\w+)$',  # maintain legacy module usage instead of menu
     # case property: list/detail
-    'module_sheet': r'^(.+):(list|detail)$',
+    'module_sheet': r'^(.+):(list|detail|case_search_display|case_search_hint)$',
```

Comment:
I'm not entirely sure how this works, but this doesn't seem quite right because this seems to be distinguishing something on a higher level (list/long or short detail i'm guessing?) and how are all the other texts matched?

Should this be done within `_get_condensed_rows`?

Reply:
`CONTEXT_REGEXS['module_sheet']` is only used in `TranslationsParser._add_module_sheet` to distinguish whether a row is of type list (row comes from the Case List tab) or type detail (row comes from Case Detail tab) and doesn't have anything to do with distinguishing long/short detail. It matches text of the format `case_property_name:property_type`. This regex hasn't been updated since we've added case search features (I assume) much later, which introduced the new types case_search_display and case_search_hint. This is an oversight because the bulk application translation uploader explicitly expects and handles for these types, so it makes sense to include them in the rows we send to Transifex as well. Hopefully this answered your question!

Reply:
Ah just to make sure i understand, so are you saying with this change, the column with header `list_or_detail` could also contain the value `case_search_display` or `case_search_hint`. So the same case property would have two rows, one for `case_search_display` and one for `case_search_hint`? That would explain this [line](https://github.com/dimagi/commcare-hq/blob/07bc788fc8dd24457a782c571957d972ed65e0a4/corehq/apps/translations/app_translations/upload_module.py#L116-L117)

I feel like that seems confusing to be doing, especially with the naming of the variables. But I would say that's outside the scope of this PR. I would suggest just updating the comment above `module_sheet`

Reply:
yep that's right. yea the naming is a little confusing but I think it's fine as is - unless I guess we plan to add more property types in the future